src/schema/v2/sale_artwork.ts

```diff
@@ -244,10 +245,20 @@ export const SaleArtworkType = new GraphQLObjectType<any, ResolverContext>({
           )
         },
       },
+      isWatching: {
+        type: GraphQLBoolean,
+        description: "Is this sale artwork being watched by a user",
+        resolve: (saleArtwork) => {
+          return Boolean(saleArtwork.isWatching)
```

Comment:
Fixme

Reply:
What needs to be fixed?

Reply:
Needed to wire this up to both the `myBids` and `watchedLotConnection`. All done 👍